src/extension/app-schema/app-schema-web/pom.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (C) 2020 - Open Source Geospatial Foundation. All rights reserved.
+    This code is licensed under the GPL 2.0 license, available at the root
+    application directory.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.geoserver.extension</groupId>
+        <artifactId>gs-app-schema</artifactId>
+        <version>2.18-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.geoserver.extension</groupId>
+    <artifactId>gs-app-schema-web</artifactId>
```

Comment:
This name is configuring. Normally modules with "web" inside have UI elements and can be removed if the Wicket GUI is tossed away. This one instead seems to contain the complex to simple conversion machinery... should have been named ``app-schema-core`` instead.